R/delineate.R

```diff
@@ -29,8 +31,8 @@
 delineate_corridor <- function(
   city_name, river_name, crs = NULL, bbox_buffer = NULL,
   initial_method = "valley", initial_buffer = NULL, dem = NULL,
-  capping_method = "direct", angle_threshold = 90, segments = FALSE,
-  riverspace = FALSE, ...
+  max_iterations = 5, capping_method = "direct", angle_threshold = 90,
```

Comment:
Shouldn't `max_iterations` be passed down to `get_corridor()`?

Reply:
Thanks for catching this!